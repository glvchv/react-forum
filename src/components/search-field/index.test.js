import React from 'react';
import renderer from 'react-test-renderer';
import SearchField from '.';


describe('Search-field component', () => {
    it('should render component', () => {
        const tree = renderer.create(<SearchField
             filterBySelect = {() => {}}
             filterPostsByWord = {() => {}}
             filterByButton = {() => {}}/>)
             .toJSON();
        expect(tree).toMatchSnapshot();
    });
});