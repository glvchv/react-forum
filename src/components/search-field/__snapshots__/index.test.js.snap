// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search-field component should render component 1`] = `
<div
  className="wrapper"
>
  <p>
    Search by category:
  </p>
  <select
    onChange={[Function]}
  >
    <option
      value="all"
    >
      See all
    </option>
    <option
      value="sports"
    >
      Sports
    </option>
    <option
      value="entertainment"
    >
      Entertainment
    </option>
    <option
      value="education"
    >
      Education
    </option>
    <option
      value="technology"
    >
      Technology
    </option>
    <option
      value="other"
    >
      Other
    </option>
  </select>
  <label>
    Search by word:
  </label>
  <input
    onChange={[Function]}
    type="text"
  />
  <button
    className="buttons"
    onClick={[Function]}
    value="newest"
  >
    Newest
  </button>
  <button
    className="buttons"
    onClick={[Function]}
    value="top"
  >
    Top rated
  </button>
</div>
`;
